File: app/src/main/java/mod/trindade/dev/preferences/example/MainActivity.java
package mod.trindade.dev.example;

import android.os.Bundle;
import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;

import com.trindade.preferences.databinding.ActivityMainPrefs;

import mod.trindade.dev.theme.ThemedActivity; 

public class MainActivity extends ThemedActivity {

    private ActivityMainPrefs binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainPrefs.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.cardSimplePreferences.setOnClickListener(view -> {
            startActivity(new Intent(this, PreferencesActivity.class));
        });
    }
}

File: app/src/minApi26/java/mod/trindade/dev/filepicker/files/activities/MainActivity.kt
package mod.trindade.dev.filepicker.files.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.FragmentContainer
import androidx.fragment.app.FragmentContainerView
import com.trindade.ware.R
import mod.trindade.dev.filepicker.files.filelist.FileItemSet
import mod.trindade.dev.filepicker.files.filelist.FileListener
import mod.trindade.dev.filepicker.files.filepicker.FilePickerCallbacks
import mod.trindade.dev.filepicker.files.fragments.FileListFragment
import mod.trindade.dev.filepicker.files.models.FileModel

class MainActivity() : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_filep)

        val fragment = supportFragmentManager.findFragmentById(R.id.fragment_container)
    }
}